raiden/network/transport/matrix/transport.py

```diff
@@ -1217,22 +1227,16 @@ def _maybe_initialize_web_rtc(self, address: Address) -> None:
                 partner_address=to_checksum_address(address),
             )
             # initiate web rtc handling
-            self._schedule_new_greenlet(self._initialize_web_rtc, address)
+            self._web_rtc_channel_inits.add(address)
+            try:
+                self._schedule_new_greenlet(self._initialize_web_rtc, address)
+            finally:
+                self._web_rtc_channel_inits.remove(address)
```

Comment:
Whats the purpose of `self._web_rtc_channel_inits` if you remove the address directly after scheduling the init process?

Reply:
Right, that's just wrong. The idea was to not schedule multiple times for the same address, but that doesn't work the way it's currently done. Thanks for noticing. This really needs to be moved into `_initialize_web_rtc`.

Reply:
Wrapped `_initialize_web_rtc` so the address stays in the set for the duration of the init.